.docker/test_all.sh

```diff
@@ -25,15 +25,19 @@ PREFIX="libreant-inst-test__"
 
 
 function cleanup {
-    for os in ${OSES[@]}; do
-        docker kill ${OSES[@]/#/"$PREFIX"} > /dev/null 2>&1
+    set +e
```

Comment:
is `set -e` ever set back? it is an important option to make scripts more robust.
Since it is just about one line, couldn't we do this instead

```sh
docker rm -f ${OSES[@]/#/"$PREFIX"} > /dev/null 2>&1  || true
```

?

Reply:
You're rigth, the error flag does not ever set back, but the `cleanup` function is only used on the script exit procedure, thus there shuldn't be any additional code executed after that cycle. Anyway I really like your solution thus I've adopted it ;)